### PR TITLE
fix(internet): add Hebrew username charMapping

### LINKED ISSUE #2776: Email addresses are unknown on fakerHE in Hebrew
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [X] I am willing to provide a PR.

### Describe the bug

When using faker.internet.email() with fakerHE, unknown email addresses are obtained, consisting of random characters. For example:
- 15t15q15f160.15c15l16215r@hotmail.com
- 15l15j15i15z15c15o3@gmail.com
- 15d16015z162_16215h16015r@yahoo.com

Suggested solution:
Add support for names in email addresses using simple pronunciation-hyphenation translation, which will allow creating more correct email addresses, even if the translation is not accurate.

Advantages:
Creating more realistic email addresses.
Improving the usability of fakerHE for Hebrew users.
effect:
This change will affect all fakerHE users who use faker.internet.email() to create Hebrew email addresses.

Code to create a translation using according to pronunciation
```

const hebrewAlphabet = {
  "א": "a",
  "ב": "b",
  "ג": "g",
  "ד": "d",
  "ה": "h",
  "ו": "v",
  "ז": "z",
  "ח": "ch",
  "ט": "t",
  "י": "y",
  "כ": "k",
  "ך": "kh",
  "ל": "l",
  "ם": "m",
  "מ": "m",
  "ן": "n",
  "נ": "n",
  "ס": "s",
  "ע": "a",
  "פ": "f",
  "ף": "ph",
  "צ": "ts",
  "ץ": "ts",
  "ק": "k",
  "ר": "r",
  "ש": "sh",
  "ת": "t",
  "ו": "v",
};

function pronounce(word) {
  let pronouncedWord = "";
  for (let i = 0; i < word.length; i++) {
    pronouncedWord += hebrewAlphabet[word[i]];
  }
  return pronouncedWord;
}

function translate(text) {
  const words = text.split(" ");
  const translatedWords = words.map(word => pronounce(word));
  return translatedWords.join(" ");
}

translate(inputText);
```

### Minimal reproduction code

```
const { fakerHE: faker } = require('@faker-js/faker');
const contacts = [];
for (let i = 0; i < 10; i++) {
    contacts.push({
      name: faker.person.fullName(),
      email: faker.internet.email(),
      phone: faker.phone.number(),
      location: faker.location.street() + " "  + faker.location.buildingNumber() + " " + faker.location.city()
    });
}
```

### Additional Context

_No response_

### Environment Info

```shell
node.js v18.16.0
```

### Which module system do you use?

- [X] CJS
- [ ] ESM

### Used Package Manager

npm

### PR DESCRIPTION
**Pull request by mordi9**
instead of getting such a result
15t15q15f160.15c15l16215r@hotmail.com
Following https://github.com/faker-js/faker/issues/2776

### CODE REVIEW

File: src/modules/internet/char-mappings.ts [hunk 1]
```diff
@@ -276,10 +276,41 @@ const farsiMapping: { [key: string]: string } = Object.fromEntries([
   ['ژ', 'zh'],
   ['ی', 'y'],
 ]);
+const hebrewMapping: { [key: string]: string } = Object.fromEntries([
+    ["א", "a"],
```
Comment (ST-DDT) [inline src/modules/internet/char-mappings.ts]: Please fix the lint errors.

**Review by ST-DDT — CHANGES_REQUESTED** [review]
You might be able to automatically fix it with: `pnpm run lint --fix`

**Comment by matthewmayer** [comment]
Here's 100 outputs:

```
'chnyt_kysry27@yahoo.com',
  'anbl.bytyvsph40@hotmail.com',
  'ravt77@gmail.com',
  'alksndr14@hotmail.com',
  'glbva27@hotmail.com',
  'ktyh_artsy68@hotmail.com',
  'tvm_bnravbn@gmail.com',
  'yalh37@gmail.com',
  'ntnal.glbva83@yahoo.com',
  'nykvls_lydvr84@yahoo.com',
  'mvryh_zkvt@hotmail.com',
  'tvbh_tshby@gmail.com',
  'rvnyt65@yahoo.com',
  'lvy94@yahoo.com',
  'alma98@gmail.com',
  'aylay71@yahoo.com',
  'dgn.knr68@gmail.com',
  'shavl57@hotmail.com',
  'ravt_avryh@gmail.com',
  'lb23@yahoo.com',
  'lyavn.bytyvsph@gmail.com',
  'ygal_gdrvn@yahoo.com',
  'chmy_bnyshral@hotmail.com',
  'shyrh88@hotmail.com',
  'yvab19@gmail.com',
  'adn.mshany@hotmail.com',
  'yrdn74@yahoo.com',
  'vyktvryh_alyab@gmail.com',
  'lshm.ashbl@yahoo.com',
  'rn.ydyn@yahoo.com',
  'shchr.gdvt12@yahoo.com',
  'ydyd33@hotmail.com',
  'yftch.ksht@gmail.com',
  'avfl_tsvk70@gmail.com',
  'yrdn.bchvr@hotmail.com',
  'yhvntn19@yahoo.com',
  'avrly.mshh@yahoo.com',
  'ym_knan@yahoo.com',
  'drvr_atsmvn43@hotmail.com',
  'shyrh25@hotmail.com',
  'nvah_avbdyh53@yahoo.com',
  'alchnn_adyny@hotmail.com',
  'syny_bram@yahoo.com',
  'rvn16@yahoo.com',
  'mrym_shr67@yahoo.com',
  'styv_dvtn@hotmail.com',
  'dnyalh0@yahoo.com',
  'shkmh59@hotmail.com',
  'yal_agrvn@hotmail.com',
  'yvbl35@hotmail.com',
  'aylay_tsmch97@hotmail.com',
  'avfk.bravr33@hotmail.com',
  'avz92@hotmail.com',
  'thylh29@yahoo.com',
  'lvy_fld44@hotmail.com',
  'nvah.bnaryh@yahoo.com',
  'yhl.shny96@yahoo.com',
  'dnyalh_kvrn@gmail.com',
  'yvgb88@yahoo.com',
  'afryl_bram93@hotmail.com',
  'flg29@hotmail.com',
  'ayylt16@gmail.com',
  'chmvtl_asys45@hotmail.com',
  'amlyh.mrvm@yahoo.com',
  'hll95@yahoo.com',
  'gl_ksfy86@gmail.com',
  'rgb78@gmail.com',
  'lbnh.advny@gmail.com',
  'chbyb.gbavn@gmail.com',
  'avhd_bnshchr81@hotmail.com',
  'ashr97@gmail.com',
  'syny30@gmail.com',
  'avfyrh_nshr@hotmail.com',
  'nytay.shbyb17@yahoo.com',
  'lyan8@hotmail.com',
  'gyvra33@yahoo.com',
  'mykl95@hotmail.com',
  'alvn.kynn@gmail.com',
  'adn24@gmail.com',
  'avmry_gl1@yahoo.com',
  'kvryn_abnvn@yahoo.com',
  'mgl.shlv53@hotmail.com',
  'shchk.bngl68@yahoo.com',
  'nchshvn_nvvh48@gmail.com',
  'lyrvy92@yahoo.com',
  'avzyh98@yahoo.com',
  'rvn13@hotmail.com',
  'lyrvy_lbary@yahoo.com',
  'smdr_chryph86@yahoo.com',
  'abytl_ntnal@gmail.com',
  'chnn79@hotmail.com',
  'rbyb_chryph98@yahoo.com',
  'alvmh_shvb@yahoo.com',
  'rvtm_mvryh34@yahoo.com',
  'nytay.mavz16@gmail.com',
  'rbkh5@yahoo.com',
  'dfnh86@gmail.com',
  'tyav_drvr40@yahoo.com',
  'shryt_frch@gmail.com',
  'ayb.aylt@gmail.com'
```

**Review by ST-DDT — APPROVED** [review]

**Review by matthewmayer — APPROVED** [review]
looks good to me, the resulting emails look better than before. they are a bit short on vowels, but that's just [how Hebrew works](https://judaism.stackexchange.com/questions/10370/why-does-the-hebrew-alphabet-not-have-letters-representing-vowels)

**Review by Shinigami92 — APPROVED** [review]

**Pull request merged**